virttest/utils_time.py

```diff
@@ -105,20 +105,20 @@ def get_timezone_list():
 
     def get_timezone_code(timezone_name):
         for value in get_timezone_list():
-            if value[1] == timezone_name:
+            if len(value) == 2 and value[1] == timezone_name:
```

Comment:
Well, the problem you met should be a bug of the serialport communication tool. It would be ok for me to make a workaround if we do not have enough time to fix it, just one question, does it work for network sessions?

Reply:
Thanks. Updated, and patch works well for network session.

 (1/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (804.80 s)
 (2/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.timedrift.shared_ntp_date.with_reboot: PASS (138.13 s)